CHANGELOG.md

```diff
@@ -2,15 +2,48 @@
 
 ## Releases
 
+### Release 2019-02-12 - Hotfix Release
+
+**Hotfix releases follow an accelerated rollout schedule - this release should be in all regions by 12am PST**
```

Comment:
For complete clarity, maybe put a date next to 12am PST? I know it's implied by the date in the release name above, but people might not pick up on that.

Reply:
fixed